attributes/default.rb

```diff
@@ -359,6 +359,15 @@
 default['datadog']['process_agent']['container_interval'] = nil
 default['datadog']['process_agent']['rtcontainer_interval'] = nil
 
+# System probe functionality settings
+default['datadog']['system_probe']['enabled'] = false
+# sysprobe_socket defines the unix socket location
+default['datadog']['system_probe']['sysprobe_socket'] = '/opt/datadog-agent/run/sysprobe.sock'
+# debug_port is the http port for expvar, it is disabled if not specified
+default['datadog']['system_probe']['debug_port'] = 0
```

Comment:
Is 0 a value understood by the system-probe as `no debug port`? Shouldn't we consider having `nil` here instead?

Reply:
Is it an interesting parameter for regular users?

Reply:
It should be 0 here as default(disabled), the comment might be misleading, I'll change it. If specified with a port it exposes expvars and pprof, the main datadog-agent could collect some metrics from this endpoint.